runtime/service/src/test/java/org/apache/polaris/service/catalog/iceberg/IcebergCatalogHandlerAuthzTest.java

```diff
@@ -1069,6 +1087,29 @@ public void testUpdateTableForStagedCreateInsufficientPermissions() {
         () -> newWrapper().updateTableForStagedCreate(TABLE_NS1A_2, new UpdateTableRequest()));
   }
 
+  @Test
+  public void testUpdateTableFallbackToCoarseGrainedWhenFeatureDisabled() {
+    // Test that when fine-grained authorization is disabled, it falls back to
+    // TABLE_WRITE_PROPERTIES
+    // This test validates that the feature flag works correctly by testing the negative case
```

Comment:
This seems to contradict the class-level javadoc comment that says that the feature is default true: `ENABLE_FINE_GRAINED_UPDATE_TABLE_PRIVILEGES (currently true).`

In this case since it's explicitly testing the "disabled" case maybe we should set the property directly in this test method?